acquisitions/src/main/resources/thunderjet/consortia/tenant-utils/consortium.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Consortium object in mod-consortia api tests
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+
+  Scenario: Create, Read, Update a consortium for positive cases
+    * def consortiumName = 'Consortium name for test'
+
+    # create a consortium
+    Given path '/consortia'
+    And request { id: '#(consortiumId)', name: '#(consortiumName)' }
+    When method POST
+    And match response == { id: '#(consortiumId)', name: '#(consortiumName)' }
```

Comment:
we need only this one call to create consortium here, all other test cases executed in separate mod-consortia module

Reply:
removed other calls